test/integration/long/execution-profile-tests.js

```diff
@@ -0,0 +1,93 @@
+"use strict";
+var assert = require('assert');
+var util = require('util');
+
+var helper = require('../../test-helper.js');
+var Client = require('../../../lib/client.js');
+var types = require('../../../lib/types');
+var utils = require('../../../lib/utils.js');
+var loadBalancing = require('../../../lib/policies/load-balancing.js');
+var DCAwareRoundRobinPolicy = loadBalancing.DCAwareRoundRobinPolicy;
+var WhiteListPolicy = loadBalancing.WhiteListPolicy;
+var ExecutionProfile = require('../../../lib/execution-profile.js').ExecutionProfile;
+
+describe('ProfileManager', function() {
+  this.timeout(120000);
+  before(helper.ccmHelper.start('2:2:2'));
+  after(helper.ccmHelper.remove);
+
+  var dc1Profile = new ExecutionProfile('default', {
+    loadBalancing: new DCAwareRoundRobinPolicy('dc1')
```

Comment:
We could add a method to check that `init()` is being called only once:
```javascript
// ...
  loadBalancing: initOnlyOnce(new DCAwareRoundRobinPolicy('dc2'))
// ...
function initOnlyOnce(policy) {
  var baseInit = policy.init;
  policy._initCalled = 0;
  policy.init = function () {
    policy._initCalled++;
    baseInit.apply(policy, arguments);
  };
  return policy;
}

// ...
client.options.profiles.forEach(function (p) {
  assert.strictEqual(p.loadBalancing._initCalled, 1);
});
```

This assertion can be interesting as `emptyProfile` uses the same lbp as `dc1Profile`.